src/utils/features.tsx

```diff
@@ -64,7 +64,7 @@ export const featuresCardsData = [
         icon: Terminal,
         iconText: <span>Linux<br />Terminals</span>,
         paragraphs: <React.Fragment>
-            <p>Gitpod instantly starts a machine in the cloud based on a Docker image. Tools that are required for your project or popular are readily installed and configured.</p>
+            <p>Gitpod instantly starts a machine in the cloud-based on a Docker image. Tools that are required for your project or popular are readily installed and configured.</p>
```

Comment:
Not sure whether this makes sense I think this was fine.
```suggestion
            <p>Gitpod instantly starts a machine in the cloud based on a Docker image. Tools that are required for your project or popular are readily installed and configured.</p>
```